docs/tutorial.md

```diff
@@ -0,0 +1,396 @@
+# The Simplest Thing That Can Possibly Work
+
+This repository template contains a lot of material, and provides a
+lot of functionality out of the box. You might feel intimidated and
+confused by the sheer number of files in it, and not know where to
+look to get what you need done.
+
+This tutorial is to walk you through the steps of getting a basic
+Hello-World-style app into production, to show you where to find the
+functionality you need and to demystify some of the content so that
+you can get the most out of it.
+
+We'll be building a python AWS Lambda handler, so to complete the
+later parts of the tutorial you'll need AWS credentials that let you
+deploy to it.
+
+I will assume familiarity with git itself.
+
+## First things first
+
+You will need certain tools to be installed in order to use the tools
```

Comment:
Could we refer or link from here to the [Prerequisites](../README.md#prerequisites) section?